### PR TITLE
Bump max supported OCP version to 4.15

### PR DESCRIPTION
**Pull request by danielerez**

### CODE REVIEW

**Review by filanov — APPROVED** [review]

File: pkg/consts/consts.go [hunk 1]
```diff
@@ -1,7 +1,7 @@
 package consts
 
 const (
-	MaxOcpVersion = "4.14" // Latest supported version (update on each release)
+	MaxOcpVersion = "4.15" // Latest supported version (update on each release)
```
Comment (filanov) [inline pkg/consts/consts.go:4]: why does max required? isn't minimal version enough?

Comment (danielerez) [reply pkg/consts/consts.go:4]: We maintain the max version for retesting the appliance on every minor version. As we might need to add additional images to oc-mirror for bootstrap, so it's not self maintained... But at least we get a clean fallback to the latest supported version by having a constant.

**Pull request merged**